### PR TITLE
fixed the typo described in issue #96

### PR DESCRIPTION
**Pull request by abhishekabramaina**
There was a typo in the section refactoring to functional style at [`Picking select elements with if`](https://dev.java/learn/refactoring-to-functional-style/foreachwithif/#picking) now I tried to fix it.

### CODE REVIEW

**Comment by abhishekabramaina** [comment]
I have signed the OCA and submitted it, and I have been waiting for approval.

**Review by ammbra — APPROVED** [review]
Looks good to me. Thank you :)

**Comment by abhishekabramaina** [comment]
@JosePaumard, could you please review the pull request? Your feedback will help the work. Thank you!

**Pull request merged**
